apps/site-landing/internals/mailjet/newsletter-form.html

```diff
@@ -0,0 +1,105 @@
+<style>
+  #container {
+    display: flex;
+    justify-content: center;
+    line-height: normal;
+    text-align: left;
+    background-color: transparent;
+    box-shadow: none;
```

Comment:
I used this file to write the HTML for the embedded widget.  It's not really necessary, and violates DRY, so removing it